Formula/h/huggingface-cli.rb

```diff
@@ -2,11 +2,11 @@ class HuggingfaceCli < Formula
   include Language::Python::Virtualenv
 
   desc "Client library for huggingface.co hub"
-  homepage "https://huggingface.co/docs/huggingface_hub/index"
+  homepage "https://huggingface.co/docs/huggingface_hub/guides/cli"
   url "https://files.pythonhosted.org/packages/91/8a/1362d565fefabaa4185cf3ae842a98dbc5b35146f5694f7080f043a6952f/huggingface_hub-0.33.0.tar.gz"
   sha256 "aa31f70d29439d00ff7a33837c03f1f9dd83971ce4e29ad664d63ffb17d3bb97"
   license "Apache-2.0"
-  revision 1
+  revision 2
```

Comment:
Let's not bump the revision; it's not necessary.

Reply:
@ZhongRuoyu fixed. But why is it not necessary? The patched version can't overwrite the existing one, can it?

Reply:
since it is a build dependency change, no need to bump (which would notify user to upgrade), and we dont need a new bottle either, as nothing would be changed.